ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/App.java

```diff
@@ -16,7 +16,7 @@
 @SpringBootApplication(scanBasePackages = {
     "uk.gov.hmcts.reform.authorisation",
     "uk.gov.hmcts.cmc.ccd.migration",
-    "uk.gov.hmcts.cmc.ccd.mapper"
+    "uk.gov.hmcts.cmc.ccd_adapter.mapper"
```

Comment:
ccd.adapter? Not sure if _ will be accepted, but for consistency pls make it `.`

Reply:
Underscore is recommended in cases where a hyphen would normally be used - so depends on if this is meant to be `ccd-adapter` or `ccd adapter` (space).
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
its already changed to `ccd.adapter` - checkstyle pulled it too 😊